webapp/views.py

```diff
@@ -178,22 +183,29 @@ def advantage_view():
                     contract["entitlements"] = entitlements
                     contract["contractInfo"][
                         "createdAtFormatted"
-                    ] = datetime.datetime.strptime(
+                    ] = datetime.strptime(
                         contract["contractInfo"]["createdAt"],
                         "%Y-%m-%dT%H:%M:%S.%fZ",
                     ).strftime(
                         "%d %B %Y"
                     )
-                    if "effectiveFrom" in contract["contractInfo"]:
-                        contract["contractInfo"][
-                            "effectiveFromFormatted"
-                        ] = datetime.datetime.strptime(
-                            contract["contractInfo"]["effectiveFrom"],
-                            "%Y-%m-%dT%H:%M:%S.%fZ",
-                        ).strftime(
-                            "%d %B %Y"
+
+                    contract["contractInfo"]["status"] = "active"
+                    if contract["contractInfo"].get("effectiveTo"):
+                        effectiveTo = datetime.strptime(
```

Comment:
This is not how we write variable names in Python - see https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles. I'll fix this and commit it if I can.